README.md

```diff
@@ -43,6 +43,7 @@ Modern HTTP-based transport supporting both direct HTTP requests and Server-Sent
 | `TRANSPORT_MODE` | Set to `streamable-http` to enable HTTP transport (legacy `http` value still supported) | `stdio` |
 | `TRANSPORT_HOST` | Host to bind the HTTP server | `127.0.0.1` |
 | `TRANSPORT_PORT` | HTTP server port | `8080` |
+| `ENDPOINT_PATH` | HTTP server endpoint path | `/mcp` |
```

Comment:
@sachinmalanki - overall looks good!

Mind changing the env var name and flag to `MCP_ENDPOINT`?